### ISSUE TITLE
[Feature Request] List of Unresolved/Resolved Suggestions in editor

### ISSUE THREAD

**Issue by bfkolle** [superdesk/superdesk-client-core#2960]
Hello,

The inline comments have a function that allows you to view unresolved and resolved comments. It would be extremely helpful if the suggestions feature had that same functionality.

Thanks.

**Comment by karelpetrak** [comment]
Hi @bfkolle Thanks for the feedback. Both suggestions (#2959 and this one) make sense. Our dev pipeline is quite pack at the moment but we'll see what we can do. The 2959 is definitely more important and we'll try to prioritize it.

**Comment by bfkolle** [comment]
@karelpetrak Thanks for looking into it!

**Comment by antonborgstrom** [comment]
I agree! This feature would be extremely useful, to be able to se all resolved/unresolved suggestions in a list.

**Comment by antonborgstrom** [comment]
@karelpetrak Is this feature something you are still considering adding?
